apps/docs/pages/docs/developer-guide/build.mdx

```diff
@@ -7,7 +7,6 @@ import { Tabs, Tab } from '../../../components/Tabs.tsx'
 1. Install [build prerequisites](#build-prerequisites) on your system
 2. `git clone https://github.com/modtree/modtree.git`
 3. `cd modtree && yarn`
-4. `yarn build`
```

Comment:
there is really no need to run build before being able to execute anything, since most of our executions are compiled JIT